### ISSUE TITLE
Parallel Tasks Example

### ISSUE THREAD

**Issue by carimura** [iron-io/docs#59]
Show how a user can run hundreds of parallel tasks to decrease the clock time.

**Comment by paddycarver** [comment]
What does "decrease the clock time" mean? Sorry, not getting what you're suggesting.

**Comment by carimura** [comment]
oh sorry I was scribbling and it was late i think.

10 tasks that take 1 minute each ran serially = 10 minutes to finish
10 tasks that take 1 minute each ran in parallel = 1 minute to finish

1 minute clock time < 10 minutes clock time

== good on ya IronWorker

**Comment by paddycarver** [comment]
Closing this as a duplicate of #66
